src/Windows/Perspex.Win32/Interop/UnmanagedMethods.cs

```diff
@@ -532,7 +532,72 @@ public enum WindowsMessage : uint
             WM_DISPATCH_WORK_ITEM = WM_USER,
         }
 
-        [DllImport("user32.dll", SetLastError = true)]
+		public enum BitmapCompressionMode : uint
+		{
+			BI_RGB = 0,
+			BI_RLE8 = 1,
+			BI_RLE4 = 2,
+			BI_BITFIELDS = 3,
+			BI_JPEG = 4,
+			BI_PNG = 5
+		}
+
+		public enum DIBColorTable
+		{
+			DIB_RGB_COLORS = 0,     /* color table in RGBs */
+			DIB_PAL_COLORS          /* color table in palette indices */
+		};
+
+		[StructLayout(LayoutKind.Sequential)]
+		public struct RGBQUAD
+		{
+			public byte rgbBlue;
+			public byte rgbGreen;
+			public byte rgbRed;
+			public byte rgbReserved;
+		}
+
+		[StructLayout(LayoutKind.Sequential)]
+		public struct BITMAPINFO
+		{
+			// C# cannot inlay structs in structs so must expand directly here
+			//
```

Comment:
Really? I wasn't aware of this limitation, and pinvoke.net suggests that it can be done: http://www.pinvoke.net/default.aspx/Structures/BITMAPINFO.html